tests/foreman/ui/test_org.py

```diff
@@ -744,30 +766,24 @@ def test_add_environment(self, env_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        env = entities.Environment(name=env_name).create_json()
-        self.assertEqual(env['name'], env_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_envs=[env_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_env'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % env_name))
-            self.assertIsNotNone(element)
+            for env_name in valid_env_names():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.